pkg/internal/windows-defender-exclusion.ps1

```diff
@@ -0,0 +1,23 @@
+# This script creates an exclusion for the given file path if the Windows Defender antivirus is running on the instance.
+# No action taken otherwise. Returns nothing in either case.
+
+# USAGE
+#    windows-defender-exclusion.ps1 [OPTIONS] <file_path>
+# OPTIONS
+#    -BinPath                 path to the file that should be excluded
+# EXAMPLES
+#    windows-defender-exclusion.ps1 "C:\k\containerd\containerd.exe"
+#    windows-defender-exclusion.ps1 -BinPath "C:\k\containerd\containerd.exe"
+
+
+param(
+    [Parameter(Mandatory=$true)] [String] $BinPath
+)
+
+# Check if the process associated with Windows Defender exists. Reference:
+# https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility?view=o365-worldwide#use-windows-powershell-to-confirm-that-microsoft-defender-antivirus-is-running
+$defenderProcessExists = (Get-Process -Name MsMpEng -ErrorAction SilentlyContinue) -ne $null
```

Comment:
you should not SilentlyContinue on error, but rather stop execution of the script

Reply:
This is the best way to check process existence according to the powershell references I've found. Getting an error is a valid, expected result. 
I guess I can explore error handling to check that if an error is thrown, is it a "process not found" PS exception? If so continue, otherwise throw the error.